MaloWBotLK/MaloWBotLK.lua

```diff
@@ -101,6 +117,8 @@ f:RegisterEvent("READY_CHECK")
 f:RegisterEvent("UNIT_SPELLCAST_START")
 f:RegisterEvent("UNIT_SPELLCAST_CHANNEL_START")
 f:RegisterEvent("UNIT_TARGET")
+f:RegisterEvent("UNIT_SPELLCAST_FAILED")
+f:RegisterEvent("UNIT_SPELLCAST_SUCCEEDED")
```

Comment:
I think using UNIT_SPELLCAST_FAILED/UNIT_SPELLCAST_SUCCEEDED will lead to some situations that is undesirable, for example on Kel'Thuzad when they strafe out of void zones they will stop the strafing if they successfully land a backstab or shred. Also if they try to cast Shred or Backstab and don't have enough energy they will get a UNIT_SPELLCAST_FAILED which will cause them to start strafing even though they're in the right position. I think using the error text "You must be behind the target" is a better hook.

Reply:
Removed these checks now.